1.9/usage/gpu/index.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
@klueska @sascala should we add a link for further reading to this presentation: https://docs.google.com/presentation/d/1FnuEW2ic5d-cpSyVOUMfUSM7WxJlZtTAAWt2dZXJ52A/edit#slide=id.g166a287193_0_42